pkg/build/api/validation/validation.go

```diff
@@ -123,13 +130,16 @@ func validateBuildSpec(spec *buildapi.BuildSpec, fldPath *field.Path) field.Erro
 	return allErrs
 }
 
-const maxDockerfileLengthBytes = 60 * 1000
+const (
+	maxDockerfileLengthBytes  = 60 * 1000
+	maxJenkinsfileLengthBytes = 60 * 1000
+)
```

Comment:
60k should be good enough for anyone?  

I think @smarterclayton introduced the original dockerfile limit.. what was the motivation?  concerns about etcd storage abuse?